routers/api/v1/repo/wiki.go

```diff
@@ -0,0 +1,501 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repo
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/convert"
+	"code.gitea.io/gitea/modules/git"
+	api "code.gitea.io/gitea/modules/structs"
+	"code.gitea.io/gitea/modules/util"
+	"code.gitea.io/gitea/modules/web"
+	wiki_service "code.gitea.io/gitea/services/wiki"
+)
+
+// NewWiki response for wiki create request
+func NewWiki(ctx *context.APIContext) {
```

Comment:
```suggestion
func NewWikiPage(ctx *context.APIContext) {
```
this naming convention on all funcs in this file would be much more understandable

Reply:
see https://github.com/go-gitea/gitea/pull/17278/commits/c908b691eb7c4b31968b9330151d9aec53758b61